### PR TITLE
Refractor dash blocks

### PR DESCRIPTION
**Pull request by nikos83**

### CODE REVIEW

File: app/views/pages/dashboard_blocks/_cars.html.haml [hunk 1]
```diff
@@ -1,3 +1,10 @@
-%ul.list-group
-  - data[:cars].each do |car|
-    %li.list-group-item.bg-secondary #{car.name}
+%h4 #{ t('.last_added')}
+
+%table.table.table-sm
+  %thead
+    %tr
+      %th{scope: 'col'} Name
```
Comment (fab-ian) [inline app/views/pages/dashboard_blocks/_cars.html.haml]: Please use everywhere 'translation'. Look at Kamil's PR https://github.com/workshop-team/motohelper/pull/57.
http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

File: app/views/pages/dashboard_blocks/_cars.html.haml [hunk 2]
```diff
@@ -1,3 +1,10 @@
-%ul.list-group
```
Comment (fab-ian) [inline app/views/pages/dashboard_blocks/_cars.html.haml]: It's not a table. It's not necessary to change it. Please don't change this block.

File: config/locales/pl.yml [hunk 3]
```diff
@@ -68,6 +69,7 @@ pl:
         mileage: 'Przebieg'
         warranty_date: 'Data gwarancji'
         note: 'Notatka'
+        closest_service: 'Najbliższy serwis'
```
Comment (fab-ian) [inline config/locales/pl.yml]: It's not an Active Record attribute(field).

File: app/views/layouts/application.html.haml [hunk 4]
```diff
@@ -7,6 +7,8 @@
     = csrf_meta_tags
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %meta{:content => "IE=edge,chrome=1", "http-equiv" => "X-UA-Compatible"}/
```
Comment (fab-ian) [inline app/views/layouts/application.html.haml]: What a mess! Feature doesn't belong to this PR and additionally this commit contains another files which are not belong to the commit too. Please don't do this again. It's horrible :)

Comment (nikos83) [reply app/views/layouts/application.html.haml]: Ok sorry I should do it separate :(

Comment (fab-ian) [reply app/views/layouts/application.html.haml]: :) Merge this PR.

Comment (fab-ian) [reply app/views/layouts/application.html.haml]: @nikos83 I wrote how manage conflicts in the first doc.
